pegasus/sites.v3/advocacy.code.org/styles/010-gotham-font.css

```diff
@@ -0,0 +1,78 @@
+
```

Comment:
Can we set these up to fall through to `sites.v3/code.org/styles`?

(I thought that happens automatically by setting "base" in `config.json`, but I could be wrong.)

Reply:
Unfortunately we do use inheritance for most assets served, but not for these combined style files (see https://github.com/code-dot-org/code-dot-org/blob/11882fb6bd7a29ecc414a190727028d8b6fc2422/pegasus/router.rb#L201 and https://github.com/code-dot-org/code-dot-org/blob/b3204960be171009ee7928a651bb2c70bbfbf66e/pegasus/helpers/page_helpers.rb#L62) but completely agreed that this duplication is not good.  Not yet sure how to do that best, but it feels like the next problem to solve here.

Reply:
So this PR's been updated to what's hopefully a better approach.  At first I tried just having advocacy.code.org load code.org/style.css, but the inclusion of Font Awesome led to CORS errors with loading a webfont cross-domain.  Now, we still generate an advocacy.code.org/style.css, but it's using the code.org source files instead of a fresh copy.  Then, there's been movement to parameterise the shared components so that they can, for example, look new on advocacy.code.org and legacy on code.org.